controllers/twophase/types.go

```diff
@@ -260,7 +260,11 @@ func updateFailedMessage(
 ) {
 	status := chaos.GetStatus()
 	status.FailedMessage = err
-	if err := r.Update(ctx, chaos); err != nil {
-		r.Log.Error(err, "unable to update chaos status")
+
+	updateError := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		return r.Update(ctx, chaos)
```

Comment:
If it conflicts at first, the retry will also result in conflict. Is it meaningful to simply retry and update here?

Reply:
Yes, You are right! I will revise and resubmit

Reply:
Done.